### PR TITLE
Added python script version of the pick_place_tutorial.cpp

### PR DESCRIPTION
**Pull request by rickstaa**
### Description

As I needed a python version of the `pick_place_tutorial`, I translated the `pick_place_tutorial.cpp` code into python. Some simplification can still be done, but I tried to mimic the `.cpp` version as closely as possible as I didn't have time to write a documentation page. I opened this pull request since I thought the script might be useful for other people.

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: doc/pick_place/scripts/pick_place_tutorial.py [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+	# Used to get information about the world and update the robot
+	# its understanding of the world.
+	move_group = robot.get_group("panda_arm")
+	planning_scene_interface = moveit_commander.PlanningSceneInterface(ns="/")
+
+	## Specify the planner we want to use ##
+	move_group.set_planner_id("TRRTkConfigDefault")
+
+	## Create a `DisplayTrajectory`_ ROS publisher to display the plan in RVIZ ##
+	display_trajectory_publisher = rospy.Publisher('/move_group/display_planned_path',
```
Comment (mlautman) [inline doc/pick_place/scripts/pick_place_tutorial.py]: please apply autopep8

File: doc/pick_place/scripts/pick_place_tutorial.py [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+	pick(move_group)
+
+	## Wait a bit ##
+	rospy.sleep(1.0)
+
+	## Place ##
+	place(move_group)
+
+if __name__ == "__main__":
+    main(sys.argv[1:])
```
Comment (mlautman) [inline doc/pick_place/scripts/pick_place_tutorial.py]: Remove sys.argv unless you really need it. rosrun causes this value to be different than if you ran it directly with the interpreter.

Comment (rickstaa) [reply doc/pick_place/scripts/pick_place_tutorial.py]: @mlautman Thanks for bringing that to my attention. Based on the [move_group python tutorial](https://ros-planning.github.io/moveit_tutorials/doc/move_group_python_interface/move_group_python_interface_tutorial.html) and the [moveit_commander.roscpp_initializer code](https://docs.ros.org/jade/api/moveit_commander/html/namespacemoveit__commander_1_1roscpp__initializer.html) I assumed the `argv` argument was needed in the roscpp_initializer.

**Comment by rickstaa** [comment]
> We really appreciate your contribution. Thank you for thinking of the community and opening this PR.
> 
> I like the idea of making this example public but I'm not sure that the `moveit_tutorials` is the right place for this to live. For one thing, there is already a c++ example script that demonstrates to the user how to get the pick and place pipeline to work so adding a python equivalent doubles the amount of maintenance without demonstrating new functionality. The other reason I am hesitant to merge this is that the pick-and-place pipeline isn't the best piece of software and there are multiple ongoing efforts to replace it. The MoveIt Task Constructor and MoveIt Grasps are far more general solutions to pick and place with moveit. If you are working on a complex project that uses MoveIt, I would recommend exploring those two libraries.

@mlautman I agree with you on that. I created a [separate repository](https://github.com/rickstaa/moveit_tutorials_python_examples) for this python example. This way people can add other python examples to this repository. Feel free to reference it somewhere in the documentation.

**Pull request closed without merging**